src/main/java/calculation/Operator.java

```diff
@@ -0,0 +1,30 @@
+package calculation;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+import java.util.stream.IntStream;
+
+public class Operator {
+	private static final String OPERATOR_ERROR_MESSAGE = "error : 입력된 연산자는 없습니다.";
+	private static final String OPERATOR_PATTERN = "[-/*+]";
+
+	public Operator() {
+	}
+
+	public static int operatorCount(Data data) {
+		return (int)Arrays.stream(data.getData())
+			.filter(i -> Pattern.matches(OPERATOR_PATTERN, i) == true)
```

Comment:
```suggestion
			.filter(i -> Pattern.matches(OPERATOR_PATTERN, i))
```
이렇게 사용해도 괜찮을 것 같네요 :)